Cloud Application Manager/Administering Your Organization/user-authentication.md

```diff
@@ -144,14 +147,15 @@ This gives your developers, operations engineers, or IT admins access to the sam
 3. Under **Authentication**, make sure LDAP is on and set up with at least one source with the conditions to sync groups.
 
 4. Turn on **LDAP Group Sync**.
+
    ![settings Auth LDAP Group Sync](../../images/cloud-application-manager/management/settings-auth-ldap-group-sync.png)
 
 5. Click **Sync** to start syncing.
-   **Note:** By default, we sync every 24 hours to get the latest group updates. To sync at any other time, click **Sync**. If a group member is deleted or moved out, they no longer have access to Cloud Application Manager workspaces and won’t be able to log in. To show or hide synchronized groups details, click on **Show Groups** or **Hide Groups**.
+   **Note:** By default, we sync every 24 hours to get the latest group updates. To sync at any other time, click **Sync**. If a group member is deleted or moved out, they no longer have access to Cloud Application Manager workspaces and won’t be able to log in. To show or hide synchronized groups details click on **Show Groups** or **Hide Groups**.
 
 ### Giving LDAP Accounts Admin Access
 
-As good practice, you should give an LDAP user in your organization administrative access to Cloud Application Manager and set aside the default administrator account to use in case of emergency. After you set up LDAP, give the LDAP user admin access as follows.
+Is good practice to give an LDAP user in your organization administrative access to Cloud Application Manager, and set aside the default administrator account to use in case of emergency. After you set up LDAP, give the LDAP user admin access as follows:
```

Comment:
```suggestion
It is a good practice to give an LDAP user in your organization administrative access to Cloud Application Manager, and set aside the default administrator account to use in case of emergency. After you set up LDAP, give the LDAP user admin access as follows:
```